### PR TITLE
fixed machine translation

### PR DESCRIPTION
**Pull request by andrhua**
not ideal though

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/russian/typescript/getters-setters/index.md [hunk 1]
```diff
@@ -1,27 +1,28 @@
...
- 
- console.log(user.fullName); 
-
-```
\ No newline at end of file
+localeTitle: Геттеры и сеттеры
+---
+# Геттеры и сеттеры
+
+TypeScript поддерживает свойства `get` и `set` . На самом деле они называются свойствами доступа (accessors). Аксессоры атрибута есть функции, связанные с получением (чтением, геттеры) или установкой (записью, сеттеры) атрибута. Объявления класса могут содержать геттер и сеттер как вместе, так и поразнь.
```
Comment (greggubarev) [inline guide/russian/typescript/getters-setters/index.md]:
```suggestion
TypeScript поддерживает свойства `get` и `set` . На самом деле они называются свойствами доступа (accessors). Аксессоры атрибута есть функции, связанные с получением (чтением, геттеры) или установкой (записью, сеттеры) атрибута. Объявления класса могут содержать геттер и сеттер как вместе, так и поразнь.
```

Comment (greggubarev) [inline guide/russian/typescript/getters-setters/index.md]:
```suggestion
TypeScript поддерживает свойства `get` и `set` . На самом деле они называются свойствами доступа (accessors). Аксессоры атрибута есть функции, связанные с получением (чтением, геттеры) или установкой (записью, сеттеры) атрибута. Объявления класса могут содержать геттер и сеттер как вместе, так и поразнь.
```

**Review by greggubarev — CHANGES_REQUESTED** [review]
поразнь =>порознь plz

File: guide/russian/typescript/getters-setters/index.md (hunk 1 above)
Comment (andrhua) [reply guide/russian/typescript/getters-setters/index.md]:
```suggestion
Объявления класса могут содержать геттер и сеттер как вместе, так и порознь.
```

**Review by greggubarev — APPROVED** [review]

**Review by greggubarev — APPROVED** [review]
LGTM 👍 :dart:

Thank you for the PR ✨ 🎉
 We're happy to accept these changes and look forward to your future contributions :clap:

Happy coding! :octocat:

**Pull request merged**